Dockerfile

```diff
@@ -56,7 +55,8 @@ LABEL name="instana-agent-operator" \
       io.openshift.tags="" \
       io.k8s.description="" \
       com.redhat.build-host="" \
-      com.redhat.component=""
+      com.redhat.component="" \
+      org.opencontainers.image.authors="Instana, support@instana.com"
```

Comment:
Would this be equivalent to the `MAINTAINER`?

Reply:
yes, MAINTAINER is deprecated according to the Dockerfile reference: https://docs.docker.com/reference/dockerfile/#maintainer-deprecated